compute_tools/src/bin/compute_ctl.rs

```diff
@@ -147,6 +150,17 @@ fn main() -> Result<()> {
     let mut state = compute.state.lock().unwrap();
     let pspec = state.pspec.as_ref().expect("spec must be set");
     let startup_tracing_context = pspec.spec.startup_tracing_context.clone();
+
+    // Record for how long we slept waiting for the spec.
+    state.metrics.sleep_ms = Utc::now()
```

Comment:
`sleep_ms` looks too generic to me, because we may have other places where sleep is used.

maybe something  like `waiting_for_spec_ms` ?

Reply:
Yeah, had the same concern, changed it to `wait_for_spec_ms` (just because other metrics are not named in 'continuing' manner, i.e. `sync_safekeepers_ms`, kinda operation name)